### ISSUE TITLE
Please export the hosted_zone_id attribute on the aws_rds_cluster resource.

### ISSUE THREAD

**Issue by ruebrenda** [terraform-providers/terraform-provider-aws#3266]
### Affected Resource(s)
- aws_rds_cluster

### Debug Output
`Resource 'aws_rds_cluster.rds_cluster' does not have attribute 'hosted_zone_id' for variable 'aws_rds_cluster.rds_cluster.hosted_zone_id'`

### Steps to Reproduce
Please list the steps required to reproduce the issue, for example:
1. `terraform plan`

```
resource "aws_route53_record" "rds" {
  count = "${length(var.hosted_zone_ids)}"

  zone_id = "${var.hosted_zone_ids[count.index]}"
  name    = "${var.dns_name}"
  type    = "A"

  alias {
    name    = "${aws_rds_cluster.rds_cluster.endpoint}"
    zone_id = "${aws_rds_cluster.rds_cluster.hosted_zone_id}"

    evaluate_target_health = false
  }
}

```

**Comment by bflad** [comment]
@ruebrenda does RDS support Route 53 alias resource records? The [Route 53 documentation](https://docs.aws.amazon.com/Route53/latest/DeveloperGuide/routing-to-rds-db.html) still lists CNAME as the solution for referencing RDS endpoints.

**Comment by ruebrenda** [comment]
@bflad RDS cluster and single instance both support Route53 alias records. I'm able to create single instance alias records in terraform using the `hosted_zone_id` exported for `aws_db_instance`. I've successfully created alias records for `aws_rds_cluster` using the AWS CLI, so I know it works.

**Comment by bflad** [comment]
Support for this has been merged into master via #3267 and will be released in v1.10.0 of the AWS provider, in about 2 weeks. 👍
